### PR TITLE
0.8.0 changelog

### PR DESCRIPTION
**Pull request by arnikola**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,14 @@ As a result of this change, M3DB will allocate significantly less mmaps, but wil
 
 Operators may need to tune their kernel configuration to allow a higher number of open file descriptors. Please follow our [Kernel Configuration Guide](http://m3db.github.io/m3/operational_guide/kernel_configuration/) for more details.
 
+## New Features
+
+- **M3DB**: Add dedicated aggreagion endpoint (#1463, #1502, #1483)
+- **M3DB**: Add coordinator capability to downsample using remote m3aggregator (#1403)
```
Comment (benraskin92) [inline CHANGELOG.md]: `M3DB` => `M3Coordinator`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,14 @@ As a result of this change, M3DB will allocate significantly less mmaps, but wil
...
+
+## Performance
+
+- **M3Query**: Tag completion endpoints (Prometheus `match[]`, `/series`, and Graphite `find` endpoints) now use the new dedicated aggregation endpoint (#1481)
```
Comment (benraskin92) [inline CHANGELOG.md]: maybe `(Prometheus (`match[]`, `/series`) and Graphite (`find`) endpoints)` instead

File: CHANGELOG.md [hunk 3]
```diff
@@ -26,8 +34,8 @@ Operators may need to tune their kernel configuration to allow a higher number o
 
 ## Performance
 
-- **M3DB**: Use a single results object for merging postings lists across blocks, rather than creating a result per block and discarding them immediately thereafter (#1474)
-- **M3DB**: Improvement to applying back pressure on writes with a full commitlog, which should improve recovery when dealing with spiky traffic (#1482)
+- **M3DB**: Use a single results object for merging postings lists across blocks, rather than creating a result per block and discarding them immediately thereafter. This should reduce memory utilization significantly for queries that look back over a large number of blocks (#1474)
```
Comment (benraskin92) [inline CHANGELOG.md]: Meant to update 0.7.3?

File: CHANGELOG.md (hunk 2 above)
Comment (arnikola) [reply CHANGELOG.md]: Nice, I like it

File: CHANGELOG.md (hunk 3 above)
Comment (arnikola) [reply CHANGELOG.md]: Yeah, i missed some comments left on the 0.7.3 changelog unfortunately

File: CHANGELOG.md (hunk 1 above)
Comment (arnikola) [reply CHANGELOG.md]: Oops, was meant to update this to a different thing

**Pull request merged**